client/stylesheets/shared/_gutenberg-components.scss

```diff
@@ -22,3 +22,7 @@ allows Woo themed components based on the config found in postcss.config.js
 @import 'gutenberg-components/tab-panel/style.scss';
 @import 'gutenberg-components/guide/style.scss';
 @import 'gutenberg-components/animate/style.scss';
+
+:root {
```

Comment:
Note that this change is borrowed from another PR that fixes issues with wp components. I can remove this before merge, but its going to be easier to test visuals with it.

Reply:
Just a note, I'm going to remove this now and remerge `main` because we have a fix for this.